### ISSUE TITLE
Wishlist: redirect `alephium.org/discord` to whatever the current invite link to discord is

### ISSUE THREAD

**Issue by Sakrecoer** [alephium/www#99]
As the title suggest: We've been in the situation before, where the invite link that we publish everywhere on the interwebs was revoked. Albeit because of a mistake (cough...cough..), still a painful event that could happen again. After all the control of those links is in the hands of Discord. 

An easyfix to this would be to have an URL on our website that simply acts as a redirect to the current discord invite link, and use that everywhere needed. This way, if the invite link would go broke for reason XYZ, all we'd have to do is to edit the link in the code of the webpage and all published links arround the web would still be valid. I propose to have `alephium.org/discord` be this redirection as it would also be easier to remember when people ask for this link :) 

This would also safeguard against the unlikely event where network effect would dictate us to chose a different chat-service, or if Discord would suffer the MySpace death.

Thank you for attending my TED talk. :purple_heart:

**Comment by Sakrecoer** [comment]
i forgot to say "Pwetty Pwease"! :pray: :slightly_smiling_face:

**Comment by nop33** [comment]
Hey @Sakrecoer , sorry, this has fallen quite back on my backlog, with the desktop wallet release, the extension building, and the mobile wallet being higher priority. Seeing that this is important for you I took an hour to investigate approaches, and I implemented a simple one.

alephium.org/discord now redirects to the same URL as the one set in the "Follow us" section. You only need to change it there from now on.

Here's a preview: https://www2.alephium.org/discord

If you're happy with the result, I'll merge and publish.
